static/src/stylesheets/module/_adslot.scss

```diff
@@ -413,11 +413,12 @@
 }
 .ad-slot--im {
     float: left;
-    margin: 5px $gs-gutter $gs-baseline 0;
+    width: 0;
+    height: 0;
 
-    @include mq($until: mobileLandscape) {
-        margin-bottom: $gs-baseline / 2;
-        margin-right: $gs-gutter / 2;
+    &.ad-slot--rendered {
```

Comment:
If adverts don't even appear in IE8, could we just use the `:not` selector to hide un-rendered IM slots?